pam/internal/adapter/utils.go

```diff
@@ -65,6 +73,55 @@ func isSSHSession(mTx pam.ModuleTransaction) bool {
 	return isSSHSessionValue
 }
 
+// GetPamTty returns the file to that is used by PAM tty or stdin.
+func GetPamTty(mTx pam.ModuleTransaction) (tty *os.File, cleanup func()) {
```

Comment:
Let's please replace all instances of `Tty` in function and variable names with `TTY` (see https://go.dev/wiki/CodeReviewComments#initialisms).

Reply:
Oh I was actually doing it in my initial version... Then I thought we wanted to keep the style. But I agree, let's do it!